codegenerator/cli/src/config_parsing/validation.rs

```diff
@@ -113,8 +113,8 @@ pub fn validate_names_valid_rescript(
     Ok(())
 }
 
-pub fn validate_deserialized_config_yaml(deserialized_yaml: &HumanConfig) -> anyhow::Result<()> {
-    if !is_valid_postgres_db_name(&deserialized_yaml.name) {
+pub fn validate_deserialized_config_yaml(evm_config: &HumanConfig) -> anyhow::Result<()> {
```

Comment:
probably what will be nice in future is to always reference the type from the module scope. Ie, `&evm::HumanConfig` etc.

Reply:
The problem with `evm::HumanConfig` is that the submodule `evm` exists in many different files. Because of this, I started aliasing `HumanConfig as EvmConfig` in the system_config file